go-client/client/tablehandle.go

```diff
@@ -0,0 +1,291 @@
+package client
+
+import (
+	"context"
+	"log"
+
+	"github.com/apache/arrow/go/v8/arrow"
+
+	tablepb2 "github.com/deephaven/deephaven-core/go-client/internal/proto/table"
+	ticketpb2 "github.com/deephaven/deephaven-core/go-client/internal/proto/ticket"
+)
+
+// A reference to a table stored on the deephaven server.
+//
+// It should be eventually released using Release once it is no longer needed.
+type TableHandle struct {
+	client   *Client
+	ticket   *ticketpb2.Ticket
+	schema   *arrow.Schema
+	size     int64
+	isStatic bool
+}
+
+func newTableHandle(client *Client, ticket *ticketpb2.Ticket, schema *arrow.Schema, size int64, isStatic bool) *TableHandle {
```

Comment:
Doc.  As a reader, it is not obvious what things like `size` or `isStatic` do.  I shouldn't have to read the whole file to know.